<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharedFilesDialog</class>
 <widget class="QWidget" name="SharedFilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>402</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="remoteframe">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeView" name="remoteDirTreeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="autoExpandDelay">
          <number>0</number>
         </property>
         <property name="itemsExpandable">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="localframe">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeView" name="localDirTreeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="downloadButton">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Download selected</string>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/download16.png</normaloff>:/images/download16.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>391</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="checkButton">
       <property name="text">
        <string>check files</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="hashLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="titleBarFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="titleBarPixmap">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/images/fileshare24.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleBarLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="filterLabel">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="images.qrc">:/images/find-16.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filterPatternLineEdit">
          <property name="toolTip">
           <string>Search files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterStartButton">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Start Search</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterClearButton">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Reset</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
 border-image: url(:/images/closenormal.png) 
}
                                
QPushButton:hover 
{
border-image: url(:/images/closehover.png) 
}
                                
QPushButton:pressed  {
border-image: url(:/images/closepressed.png) 
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="viewType_CB">
        <item>
         <property name="text">
          <string>Tree view</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Flat view</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="indicatorCBox">
        <item>
         <property name="text">
          <string>All</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/view_calendar_list.png</normaloff>:/images/view_calendar_list.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>One day old</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/view_calendar_day.png</normaloff>:/images/view_calendar_day.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>One Week old</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/view_calendar_week.png</normaloff>:/images/view_calendar_week.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>One month old</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/view_calendar_month.png</normaloff>:/images/view_calendar_month.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="splittedButton">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Split View</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/view_split_top_bottom.png</normaloff>:/images/view_split_top_bottom.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="remoteButton">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Friends Folders</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/friendsfolder24.png</normaloff>:/images/friendsfolder24.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="localButton">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>My Folders</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/folder16.png</normaloff>:/images/folder16.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
